app/controllers/plans_controller.rb

```diff
@@ -88,9 +88,14 @@ def read_exported_sources
   end
 
   def cleanup
+    log_file = File.basename(Rails.configuration.x.terraform_log_filename, '.*')
```

Comment:
is this `.*` not to broad? could we perhaps minimize it to more  specific logs thing? like `.log*` whatever postfix

Reply:
This method is used just to get the file name without the extension, as this is what I want (the logs files are expanded as `ruby-terraform.log`, ruby-terraform-1.log`, etc. So with the base name, I can create a pattern to avoid removing them
https://apidock.com/ruby/File/basename/class